modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
++
+[source,yaml]
+----
+...
+hub:
+  params:
+    - name: enable-devconsole-integration
+      value: "true"
+...
+----
+// https://github.com/tektoncd/operator/pull/569 
+
+* With this update, the `operator-config.yaml` cong map enables display of the Operator version as part of the output of the `tkn version` command.
+// https://github.com/tektoncd/operator/pull/563
+
+* With this update, the version of the `argocd-task-sync-and-wait` tasks is modified to `v0.2`.
+// https://github.com/tektoncd/operator/pull/642
+
+* With this update to the `TektonConfig` CRD, the `oc get tektonconfig` command displays the OPerator version.
+// https://github.com/tektoncd/operator/pull/644
+
+* With this update, service monitor is added to the Triggers metrics.
+// https://github.com/tektoncd/operator/pull/635
+
+[id="hub-new-features-1-7_{context}"]
+=== Hub
+
+[IMPORTANT]
+====
+Tekton Hub on cluster is a Technology Preview feature.
+====
+
+Tekton Hub helps you discover, search, and share reusable tasks and pipelines for your CI/CD workflows. A public instance of Tekton Hub is available at link:https://hub.tekton.dev/[hub.tekton.dev]. 
+
+Staring with {pipelines-title} 1.7, cluster administrators can also install and deploy a custom instance of Tekton Hub on enterprise clusters. You can curate a catalog with reusable tasks and pipelines specific to your organization.
+
+[id="chains-new-features-1-7_{context}"]
+=== Chains
+
+[IMPORTANT]
+====
+Tekton Chains is a Technology Preview feature.
+====
+
+Tekton Chains is a Kubernetes Custom Resource Definition (CRD) controller. You can use it to manage the supply chain security of the tasks and pipelines created using {pipelines-title}.
+
+By default, Tekton Chains observes all task run executions in your {product-title} cluster. When the task runs complete, Chains takes a snapshot of the task runs. It then converts the snapshot to one or more standard payload formats, and finally signs and stores all artifacts.
+
+Tekton Chains supports the following features:
+
+* You can sign task runs, task run results, and OCI registry images with cryptographic key types and services such as `cosign`.
+
+* You can use attestation formats such as `in-toto`.
+
+* You can securely store signatures and signed artifacts using OCI repository as a storage backend.
+
+[id="pac-new-features-1-7_{context}"]
+=== Pipelines as Code (PAC)
+
+[IMPORTANT]
+====
+Pipelines as Code is a Technology Preview feature.
+====
+
+With Pipelines as Code, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports status.
+
+Pipelines as Code supports the following features:
+
+* Pull request status. When iterating over a pull request, the status and control is executed on the platform hosting the Git repository.
```

Comment:
Executed => performed or runs? Multiple instances.

Reply:
```suggestion
* Pull request status. When iterating over a pull request, the status and control are performed on the platform hosting the Git repository.
```
- "is" => "are" for multiple subjects.
- This phrase is still kind of odd. Does one really execute (or perform) status and control?

Reply:
Pull request status. When iterating over a pull request, the status and control of the pull request is **exercised** on the platform hosting the Git repository.